src/backend/pipeline/modules/mosaicing.py

```diff
@@ -70,3 +77,28 @@ def run(self, data: Data) -> Data:
             
         # Run the next module
         return super().run(data)
+    
+    def process(self, img: Mat) -> (np.ndarray, Mat):
+        """
+        Extract and process information from the orthomosaic image.
+
+        Args:
+            img: a Mat object containing the stitched image
+        
+        Returns:
+            tuple consisting of a np.ndarray and a new Mat 
+            representation
+        """
+        # collapse channels to ensure all are 0
+        collapsed = np.sum(img.get(), axis=2)
+        # create the mask
+        mask = np.where(collapsed == 0.0, 1, 0)
```

Comment:
Do you think this will work well enough? I was thinking about something like
```
extract masks
cluster masks (BFS) 
delete masks that don't touch the image edge
set alpha channel to 1 for the remaining masks
```

Reply:
yeah I guess there is a small probability of having all three channels set to 0 in the middle of the image. So the BFS option would be the correct one, but sounds like an overkill. However, if you really want this, I can do it pretty fast.

Reply:
The problem is if you're running mosaicing on grayscale images, then the chance is not that low. We might also have some shenanigans when we're calibrating the data in the preprocessing module (e.g. doing radiometric correction), and a set of values are shifted to under 0 and are clamped at 0. 

I would do it but if you think it's an overkill feel free to resolve 🚀

Reply:
We can put it in a should have or move it in a separate issue for the next sprint?

Reply:
Sure